lib/spy_glass/registry/nyc-311-potholes.rb

```diff
@@ -0,0 +1,57 @@
+require 'spy_glass/registry'
+
+query = {
+  '$limit' => 1000,
+  '$order' => 'created_date DESC',
+  '$where' => <<-WHERE.oneline
+    created_date >= '#{7.days.ago.iso8601}' AND
+    longitude IS NOT NULL AND
+    latitude IS NOT NULL AND
+    descriptor = 'Pothole' AND
+    unique_key IS NOT NULL
+  WHERE
+}
+
+opts = {
+  path: '/nyc-311-potholes/',
+  cache: SpyGlass::Cache::Memory.new(expires_in: 300),
+  source: 'https://data.cityofnewyork.us/resource/erm2-nwe9.json?'+ Rack::Utils.build_query(query)
+}
+
+time_zone = ActiveSupport::TimeZone['Eastern Time (US & Canada)']
+SpyGlass::Registry << SpyGlass::Client::Socrata.new(opts) do |collection|
+  features = collection.map do |item|
+    time = Time.iso8601(item['created_date']).in_time_zone(time_zone).strftime("%m/%d  %I:%M %p")
+    city = item['city'].capitalize
+
+    title =
+      case item['address_type']
+      when 'ADDRESS'
+        "#{time} - #{item['descriptor']} at #{item['incident_address'].titleize} in #{city}."
```

Comment:
This is the current text on the overall 311, I think it's way too verbose

https://github.com/BetaNYC/citygram-services-nyc/blob/master/lib/spy_glass/registry/nyc-311.rb#L30